### DISCUSSION TITLE
Does this mod work with 4.36.11.4679?

### DISCUSSION THREAD

**Discussion by xblackedemonx** [gtxaspec/wz_mini_hacks#567] [Q&A] [answered]
Hi, I noticed that the description was updated 5 days ago and now it says _Update to the latest stable or beta firmware, this mod should still work! (most features!)_
Before it would say not to update passed 4.36.9.139.

Can we install 4.36.11.4679 with this mod now?

**Comment by gtxaspec** [comment, marked as answer]
wz_mini will work with versions beyond .139, you can get a root shell, but any `libcallback` functions will not work, ex. rtsp video streaming, any `cmd` commands.

**Reply by xblackedemonx** [reply]
Thank you for the quick reply!
